modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts

```diff
@@ -108,9 +109,19 @@ async function parseI3SNodeGeometry(arrayBuffer: ArrayBuffer, tile: Tile = {}) {
 
     for (const key in decompressedGeometry.loaderData.attributes) {
       const dracoAttribute = decompressedGeometry.loaderData.attributes[key];
-      if (dracoAttribute.name === 'POSITION') {
-        attributes.position.metadata = dracoAttribute.metadata;
-        break;
+
+      switch (dracoAttribute.name) {
+        case 'POSITION': {
+          attributes.position.metadata = dracoAttribute.metadata;
+          break;
+        }
+        case 'feature-index': {
+          attributes.id.metadata = dracoAttribute.metadata;
+          break;
+        }
+        default: {
+          break;
```

Comment:
`break` in `default` doesn't have sense because `default` is last option.

Reply:
EsLint ask for for something in default case.